tests/docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: "3.8"
```

Comment:
Dockerfile to run tests against it, not sure how/when we want to run this

Reply:
If you are asking about CI, we can do it like we do it with `status-go`:
https://github.com/status-im/status-go/blob/070ffd9f5c455e2e8de4ef5ac17b39bb6064a9ef/_assets/ci/Jenkinsfile.tests#L64-L76
But we don't need a Docker Compose for that.